#goal definition
float64 target_lat			# Info of current and target latitudes and longitudes
float64 target_long
---
#result definition
uint8 nav_state_type
uint8 NAV_STATE_TYPE_SUCCESS = 1
uint8 NAV_STATE_TYPE_ERROR = 0
uint8 NAV_STATE_TYPE_COMPLETE = 2
---
#feedback